src/bot/commands/approve.ts

```diff
@@ -395,7 +493,7 @@ export default class ApproveCommand extends Command {
     const jokeMessage = await message.edit({ embeds: [embed] });
     await jokeMessage.reactions.removeAll();
 
-    message.client.stickys.reload();
+    await message.client.stickys.reload();
```

Comment:
Bien que ce soit asynchrone c'est très lourd en terme de fonctionnement donc on préfère l'executer dans un thread différent (donc sans await) 
```suggestion
    message.client.stickys.reload();
```